paasta_tools/deployment_utils.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
I do have future plans for this file. I do intend to relocate things out of utils and fill up this one, but those moves are more "mechanical" and I didn't want to clutter up the review.